crates/swift-bridge-macro/tests/ui/incorrect-return-type.stderr

```diff
@@ -0,0 +1,23 @@
+error[E0308]: mismatched types
+ --> tests/ui/incorrect-return-type.rs:6:1
+  |
+6 | #[swift_bridge::bridge]
```

Comment:
We want to use the function's return type's span such that in

```rust
fn foo () -> SomeType;
```

the `SomeType` would get underlined.

---

To accomplish this you can change this https://github.com/chinedufn/swift-bridge/blob/54b9c09337701dd7580010d1d5ef6527b855ee98/crates/swift-bridge-ir/src/parsed_extern_fn/to_extern_c_fn.rs#L36

to something like:

```rust
        let ret_span = self.sig.output.span();
        let ret = self.rust_fn_sig_return_tokens(swift_bridge_path, types);
        let ret = quote_spanned! {ret_span=> #ret};
```

Reply:
I have tried that spanning method, but it does not produce the spanned error output. I will try to investigate the reason why.